src/test/test-path.c

```diff
@@ -102,11 +102,20 @@ static void _check_states(unsigned line,
                          service_state_to_string(service->state),
                          service_result_to_string(service->result));
 
+                if (service->state == SERVICE_FAILED)
```

Comment:
maybe filter out EXIT_CGROUP fialures explicitly and no others?

I figure we should at least try to invoke our tests in a delegated scope btw, if we can... i.e. make any test that wants to do cgroup stuff ask for a scope first

Reply:
I posted #17052 for the scope allocation logic now. ptal